### DISCUSSION TITLE
Quick Create in related module

### DISCUSSION THREAD

**Discussion by saba1996** [YetiForceCompany/YetiForceCRM#16810] [General]
hello, In which file do I find the code for the quick create menu for the related module? 

The blocks are not in sequence as shown in the settings

**Comment by rskrzypczak** [comment]
[/modules/Vtiger/views/QuickCreateAjax.php](https://github.com/YetiForceCompany/YetiForceCRM/blob/f62d703908f3b8cd74caae0151f0e23c2e63710f/modules/Vtiger/views/QuickCreateAjax.php)
